src/components/DatePicker.vue

```diff
@@ -504,9 +511,27 @@
           return D + ['th', 'st', 'nd', 'rd'][D % 10 > 3 ? 0 : (D - D % 10 !== 10) * D % 10];
         }
       };
+      let checkinDate = this.formatDate(new Date(this.checkIn) , 'YYYYMM');
```

Comment:
Please extract this whole block into a new method and just call it on `beforeMount`

Reply:
it removed